docs/content/en/contribute/docs/linters_requirements/_index.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
please adjust the folder naming to kebab case: `linter-requirements`
don't forget to adjust the link the contributing guide